seqr/utils/social_auth_pipeline.py

```diff
@@ -37,3 +37,9 @@ def log_signed_in(backend, response, is_new=False, *args, **kwargs):
     logger.info('Logged in {} ({})'.format(response['email'], backend.name), extra={'user_email': response['email']})
     if is_new:
         logger.info('Created user {} ({})'.format(response['email'], backend.name), extra={'user_email': response['email']})
+
+
+def log_azure_signed_in(backend, details, is_new=False, *args, **kwargs):
```

Comment:
this appears identical to `log_signed_in`, is there a reason this needs to be its own function?

Reply:
the Azure provider returns `email` in `details`, not `response` so `log_signed_in` crashes against Azure - it could be updated to handle both cases but I didn't want to alter the existing behavior in case you were relying on it...

Reply:
Ah I see. It would probably be a bit neater to have a `_log_signed_in` helper that both `log_signed_in` and `log_azure_signed_in` call to encapsulate that it is identical behavior, but this is fine as it is